css/header.css

```diff
@@ -0,0 +1,138 @@
+.header {
+  /* border: 3px solid black; */
```

Comment:
it's your portfolio code - it should be clean & pretty!
NOT ALLOWED
- coments
- duplicating code